src/EditorFeatures/CSharpTest/IntroduceUsingStatement/IntroduceUsingStatementTests.cs

```diff
@@ -143,6 +143,7 @@ void M(System.IDisposable disposable)
         using (var name = disposable)
         {
         }
+
```

Comment:
this actually fits the appropriate formatting when generating new syntax.  it didn't work before because of how we moved trivia around.